experimental/packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -52,13 +52,37 @@ export abstract class InstrumentationBase<T = any>
         'No modules instrumentation has been defined,' +
         ' nothing will be patched'
       );
+    } else {
+      this._warnOnPreloadedModules();
```

Comment:
Another small nit: the right place for this is actually before calling `RequireInTheMiddle` in `enable` function.

If `this._config.enabled` is truthy then it's the same, but calling`_warnOnPreloadedModules` in `enable` will also cover other scenarios.

Reply:
Done